integration/gotests/gotest_test.go

```diff
@@ -75,9 +76,9 @@ func testPkgs(t *testing.T) []string {
 		"github.com/u-root/u-root/pkg/tss",
 		"github.com/u-root/u-root/pkg/syscallfilter",
 	}
-	if qemu.GuestArch() == qemu.ArchArm64 {
-		blocklist = append(
-			blocklist,
+	switch qemu.GuestArch() {
+	case qemu.ArchArm64:
+		blocklist = append(blocklist,
 			"github.com/u-root/u-root/pkg/strace",
```

Comment:
This ought to work, I think?
syscall_linux_arm64.go is in pkg/strace?

Reply:
Let's give that a try.

arm64 qemu seems to be twice as slow as amd64 and arm. Not sure why that is.

Reply:
`../../pkg/strace/syscalls_test.go:23:28: undefined: unix.SYS_OPEN`

test itself doesn't compile. I'll leave it for another time.